src/config.ts

```diff
@@ -696,24 +701,57 @@ export default {
    * let's imagine the following SEGMENT_PRIORITIES_STEPS array:
    * [5, 11, 17, 25]
    *
-   * To link each Segments to a corresponding priority (and thus to a specific
-   * step), we have to consider the distance d between the current position and
-   * the start time of the Segment.
+   * To link each Segments to a corresponding priority number (and thus to a
+   * specific step), we have to consider the distance d between the current
```

Comment:
Is it necessary to let` distante d `here ?

Reply:
no, you're right

Reply:
Done